website/content/specification/objects/_index.md

```diff
@@ -7,20 +7,20 @@ sidenav:
 
 A Conceptual entity in the vulntology.
 
-Objects can be related to other objects, have types, and properties.  Each object, such as [vulnerability](vulnerability), can have multiple properties and/or relationships with other components. 
+Objects can be related to other objects, have types, and properties.  Each object, such as [vulnerability](vulnerability), can have multiple properties and/or relationships with other components.
 
 ## Object Types
 
-The following objects are defined by the Vulntology.
+The following objects are defined by the Vulntology. These objects are presented below in a rough hierarchical organization to illustrate how they relate to each other.
 
-- [action](action)
-- [barrier](barrier)
-- [impact](impact)
-- [impact method](impact-method)
-- [product](product)
-- [scenario](scenario)
-- [vulnerability identifier](vulnerability-identifier)
 - [vulnerability](vulnerability)
+  - [vulnerability identifier](vulnerability-identifier)
+  - [scenario](scenario)
+    - [action](action)
+    - [impact method](impact-method)
```

Comment:
impact method and impact should be indented further since they relate to action

Reply:
Resolved by the suggestion below.